### PR TITLE
[DEV-1159] Add Spinner component

### PR DESCRIPTION
**Pull request by marcobottaro**
This PR depends on #428 

#### List of Changes

Add Spinner component

#### How Has This Been Tested?

Manually

#### Screenshots (if appropriate):
image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/app/profile/personal-data/page.tsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ import { useTranslations } from 'next-intl';
 import { useUser } from '@/helpers/user.helper';
 import { InfoCardItemProps } from '@/components/atoms/InfoCardItem/InfoCardItem';
 import { InfoCard } from '@/components/molecules/InfoCard/InfoCard';
-import PageNotFound from '@/app/not-found';
 import DeleteSection from '@/components/molecules/DeleteSection/DeleteSection';
+import React from 'react';
```
Comment (kin0992) [inline apps/nextjs-website/src/app/profile/personal-data/page.tsx:9]: is this necessary?

**Review by kin0992 — COMMENTED** [review]
it's not so clear to me what is going to change this PR: there is a new Spinner component that should be used in the agreements page (according the the Jira activity linked) but I don't see the spinner in that page. am I missing something @marcobottaro?

File: apps/nextjs-website/src/app/profile/personal-data/page.tsx (hunk 1 above)
Comment (marcobottaro) [reply apps/nextjs-website/src/app/profile/personal-data/page.tsx]: Not anymore

**Comment by marcobottaro** [comment]
> it's not so clear to me what is going to change this PR: there is a new Spinner component that should be used in the agreements page (according the the Jira activity linked) but I don't see the spinner in that page. am I missing something @marcobottaro?

This PR adds the new Spinner components and uses it in the API page (for now). Once the PR is merged, the component will be used in the ProfileMenu component to cover both "agreements" page and "personal-data" page.

**Review by kin0992 — APPROVED** [review]

**Review by MarcoPonchia — APPROVED** [review]

**Pull request merged**